app/static/js/index.js

```diff
@@ -0,0 +1,14 @@
+function myFunction(x){
+    let show = document.querySelector('.right');
+    x.classList.toggle("change");
+    show.classList.toggle('show-menu');
+    show.classList.toggle('show-menu>a');
+}
+
+const humberger = document.querySelector(".hambuger-menu");
+const navMenu = document.querySelector(".right");
+
+document.querySelectorAll(".right").forEach(e => e.addEventListener("click", () => {
+    humberger.classList.remove("change");
+    navMenu.classList.remove("show-menu");
+}));
```

Comment:
- [ ] Your mobile menu is working almost fine, but it's possible to close the menu if the user clicks on the blank space below the menu items. The user can accidentally click and close the menu this way and it's not the expected behavior. Please fix this issue.
 -
<img width="700" alt="image" src="https://user-images.githubusercontent.com/61919617/189723005-8816a639-11e8-45d1-8694-08c16f071c1d.png">

Reply:
Hello @leonardodiasb 
Let I have worked on the changes you suggested.
Thanks.